api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityOverrideTestDataFactory.java

```diff
@@ -3,17 +3,30 @@
 import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
 import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
 
+import java.time.LocalDate;
+
+import static uk.gov.dhsc.htbhf.claimant.testsupport.TestConstants.OVERRIDE_UNTIL_FIVE_YEARS;
+
 public class EligibilityOverrideTestDataFactory {
 
     public static EligibilityOverride aConfirmedEligibilityOverride() {
         return EligibilityOverride.builder()
                 .eligibilityOutcome(EligibilityOutcome.CONFIRMED)
+                .overrideUntil(OVERRIDE_UNTIL_FIVE_YEARS)
+                .build();
+    }
+
+    public static EligibilityOverride aConfirmedEligibilityWithUntilDate(LocalDate overrideUntil) {
+        return EligibilityOverride.builder()
+                .eligibilityOutcome(EligibilityOutcome.CONFIRMED)
+                .overrideUntil(overrideUntil)
                 .build();
```

Comment:
Is it worth having a default builder method yet which can be reused for all these methods?